admin/module/catalog-admin-module-sources/src/main/webapp/js/view/ConfigurationEdit.view.js

```diff
@@ -121,7 +121,7 @@ define([
             _.each(this.collectionArray.models, function(model) {
                 values.push(model.get('value'));
             });
-            this.configuration.get('properties').set(this.model.get('id'), values);
+            this.configuration.get('properties').set(this.model.get('id'), values.length === 0 ? null : values);
```

Comment:
I updated the check in ConfigurationAdmin.java to check for empty arrays and reverted this line. This pull request will be closed since all changes are in ddf-admin now.